x-pack/plugins/siem/public/resolver/view/edge_line.tsx

```diff
@@ -42,12 +70,16 @@ const EdgeLineComponent = React.memo(
      */
     const screenStart = applyMatrix3(startPosition, projectionMatrix);
     const screenEnd = applyMatrix3(endPosition, projectionMatrix);
+    const [magFactorX] = projectionMatrix;
+    const { colorMap } = useResolverTheme();
+    const elapsedTime = edgeLineMetadata?.elapsedTime;
 
     /**
      * We render the line using a short, long, `div` element. The length of this `div`
      * should be the same as the distance between the start and end points.
      */
     const length = distance(screenStart, screenEnd);
+    const scaledTypeSize = calculateResolverFontSize(magFactorX, 10, 7.5);
```

Comment:
i'm not familiar w/ this function. is it in master?

Reply:
The function, no, but the logic yes. I just pulled out logic from here into a util https://github.com/elastic/kibana/blob/aec0b976151415bee54f94aefc1a6a803264d5f0/x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx#L320-L323